apis/project_api/project_api/api/disagg.py

```diff
@@ -54,6 +53,9 @@ def get_ensemble_disagg():
     )
     server.app.logger.debug(f"Request parameters {project_id}, {station_id}, {im}")
 
+    # Get Disagg return periods
+    rps = utils.get_project(version_str, project_id).disagg_rps
+
     # Load the data
     ensemble_disagg, metadata_df, src_png_data, eps_png_data = utils.load_disagg_data(
```

Comment:
Maybe change these variables as they are now lists of ensemble_disaggs not a single object, same with the metadata_df